roles/optional-operators-subscribe/tasks/main.yml

```diff
@@ -34,6 +34,6 @@
   always:
   - name: "Store the result in a JSON file"
     copy:
-      dest: "optional_operator_subscribe.json"
```

Comment:
AMT default value for two directories are the same.
```
      ARTIFACT_DIR: "{{ ARTIFACT_DIR }}"
      SHARED_DIR: "{{ ARTIFACT_DIR }}"
```

Storing json to directory with artifacts make a sense. And all of them should be included in  tarball created by your team. Do we really need to collect also data from "SHARED_DIR"

Should be one sub-directory or other one ? @dirgim You introduced the `"SHARED_DIR"`. Coudl you explan what is a difference.

Reply:
The SHARED_DIR is used by Prow jobs to share files between different testing steps since there isn't a different mechanism to relay information between steps.
This is obviously not required when running the steps in Ansible so I had decided to point the directories to the same path by default when I updated the script to the version that introduced usage of the SHARED_DIR.
I'm not entirely satisfied with this solution as it looks weird in the ansible output, so will address it in an upcoming PR.
This should not hold up this PR as it's unrelated to it, though.

Reply:
Thank you for explanation